model/stories_test.go

```diff
@@ -67,10 +69,17 @@ func TestGetStoryByID(t *testing.T) {
 		db, cleanUp := setupDBConnection(t, dbConfig)
 		defer cleanUp(t)
 
+		// We need to first create a user due to the foreign key constraint
+		user := User{
+			Username:      "testMultipleStoriesAuthor",
+			LoginProvider: userenums.LoginProvider(rand.Int31()),
+		}
+		_ = CreateUser(db, &user)
+
```

Comment:
should we just have one test to test the new batch function. can be just the same like this but with batch creation.

Reply:
I was thinking of waiting until #54 before creating the tests. What do you think?